src/core/net/nat64_translator.hpp

```diff
@@ -174,11 +254,16 @@ class Translator : public InstanceLocator, private NonCopyable
 
         void       Touch(TimeMilli aNow) { mExpiry = aNow + kAddressMappingIdleTimeoutMsec; }
         InfoString ToString(void);
+        void       CopyTo(otNat64AddressMapping &aMapping) const;
+
+        uint64_t mId; // The unique id for a mapping session.
```

Comment:
Why do we need this `mId`?

Reply:
mId is used to distinguish different sessions with same (IP4, IP6) pair, this may happen when one mapping for one client is expired, and then the client becomes active again and is assigned the same IP4 address.

In the above case, we need to distinguish the two sessions so the counters can be used correctly.